Hestia/Hestia/frontend/SettingsScreen/UITableViewControllerContacts.cs

```diff
@@ -0,0 +1,71 @@
+using Foundation;
+using System;
+using UIKit;
+using MessageUI;
+
+
+namespace Hestia
+{
+    public partial class UITableViewControllerContacts : UITableViewController
+    {
+        MFMailComposeViewController mailController;
+
+        public UITableViewControllerContacts(IntPtr handle) : base(handle)
+        {
+        }
+
+        //Opinion
+        partial void UIButton105005_TouchUpInside(UIButton sender)
+        {
+            if(MFMailComposeViewController.CanSendMail){

```

Comment:
There is a lot of duplicated code in these three methods. You should extract a methods from this. Also, "this is a test" should still be changed to something more informative I guess?